ftplugin/ledger.vim

```diff
@@ -14,7 +14,6 @@ let b:undo_ftplugin = "setlocal ".
                     \ "include< comments< commentstring< omnifunc< formatprg<"
```

Comment:
Can you also remove `foldmethod<` from `b:undo_ftplugin`?

Reply:
TIL what `b:undo_ftplugin` does ^^ Thank you, updated.